packages/dashboard/src/components/csvDownloadButton/getDescribedTimeSeries.tsx

```diff
@@ -0,0 +1,35 @@
+import { DescribeTimeSeriesCommand, IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
+import invariant from 'tiny-invariant';
+
+const isEnabled = (input?: string): input is string => Boolean(input);
+
+export const getDescribedTimeSeries = async ({
+  client,
+  assetId,
+  propertyId,
+  alias,
+}: {
+  client: IoTSiteWiseClient;
+  assetId?: string;
+  propertyId?: string;
+  alias?: string;
+}) => {
+  invariant(
+    isEnabled(alias) || (isEnabled(assetId) && isEnabled(propertyId)),
+    'Expected alias or assetID+propertyID to be defined as required by the enabled flag.'
+  );
+
+  const params = { assetId, propertyId, alias };
+  const command = new DescribeTimeSeriesCommand(params);
+
+  try {
+    const data = await client.send(command);
+    return { data, isSuccess: true };
+  } catch (error) {
+    console.error(`Failed to get latest values. Error: ${error}`);
```

Comment:
you probably want to log the error in all 3 cases ?

Reply:
im following the same error logging as the ones found the resource explorer
which have the following definitions
```
#handleError(error: unknown): never {
    console.error(`Failed to get asset models. Error: ${error}`);
    console.info('Request input:');
    console.table(this.#command.input);

    throw error;
}
```